test/wdio/slot-fallback-with-textnode/cmp.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { h } from '@stencil/core';
+import { render, waitForChanges } from '@wdio/browser-runner/stencil';
+
+describe('slot-fallback-with-textnode', function () {
+  beforeEach(() => {
+    render({
+      template: () => <my-component></my-component>,
+    });
+  });
+
+  it('should hide fallback content when provided slot is text node', async () => {
+    await expect($('.container')).toHaveText(expect.stringContaining('DEFAULT'))
```

Comment:
Can we be more strict here? If there is a new line added I recommend to add a `{ trim: true }` as second parameter
```suggestion
    await expect($('.container')).toHaveText('DEFAULT')
```